devices/schneider_electric.js

```diff
@@ -981,4 +983,27 @@ module.exports = [
                 .withUnit('A').withDescription('Instantaneous measured electrical current on phase C'),
         ],
     },
+    {
+        zigbeeModel: ['W599001'],
+        model: 'W599001',
+        vendor: 'Schneider Electric',
+        description: 'Wiser smoke alarm',
+        fromZigbee: [fz.schneider_temperature, fz.battery, fz.ias_enroll, fz.ias_smoke_alarm_1],
+        toZigbee: [],
+        ota: ota.zigbeeOTA, // local OTA updates are untested
+        exposes: [e.smoke(), e.battery_low(), e.tamper(), e.battery(), e.battery_voltage(),
+            en('local_temperature', ea.STATE).withUnit('°C').withDescription('Current temperature measured on the device'),
```

Comment:
To be in line with other devices, this should be either `temperature` or `device_temperature`.